deepchem/models/tensorgraph/layers.py

```diff
@@ -3117,13 +3178,15 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
     self.out_tensors = []
     tmp_tensor = []
     for row in range(n_alphas):
-      tmp_tensor.append(tf.reshape(subspaces[row,], [-1, subspace_size]))
+      self.out_tensors.append(tf.reshape(subspaces[row,], [-1, subspace_size]))
```

Comment:
note to self: i need to undo the alphashare diffs. I think one of the changes to master wrecked parts of this.